sound/soc/sof/topology.c

```diff
@@ -244,7 +244,7 @@ static int sof_control_load_volume(struct snd_soc_component *scomp,
 	struct sof_ipc_ctrl_data *cdata;
 
 	/* validate topology data */
-	if (le32_to_cpu(mc->num_channels) >= SND_SOC_TPLG_MAX_CHAN)
+	if (le32_to_cpu(mc->num_channels) > SND_SOC_TPLG_MAX_CHAN)
```

Comment:
This should be a separate fix in it's own PR.

Reply:
Can  you rebase and repush, I don't see why this change is still visible.

Reply:
Sorry, I will do it